dotcom-rendering/src/model/window-guardian.ts

```diff
@@ -6,7 +6,7 @@ export interface WindowGuardianConfig {
 	page: {
 		dcrCouldRender: boolean;
 		contentType: string;
-		edition: Edition;
+		edition: EditionId;
```

Comment:
same question as above - why are we leaving the key as edition in this instance?

Reply:
As above, where we're staying on window.guardian I'm hesitant to change it!